### PR TITLE
Add: integration tests for codecov testing

### PR DESCRIPTION
**Pull request by jsamoocha**
This adds 2 trivial integration tests to see if codecov diff is working.

### CODE REVIEW

**Review by lwasser — APPROVED** [review]
LGTM!! Thanks @jsamoocha and it looks like codecov is working as intended. I am curious. I setup codecov and i don't know if i need to provide you and others with permissions to see the breakdown. it's pretty cool. 

Are you able to login and see the full coverage reports? i'm just curious if everyone can see them when logged in or if i need to adjust permissions or something. an example is below. 

Screen Shot 2022-12-02 at 7 51 28 AM

**Comment by lwasser** [comment]
They changed the colors a bit so i find it harder to read now but lines with RED next to them are still uncovered lines in the code. I really like this tool!

File: stravalib/tests/integration/strava_api_stub.py [hunk 1]
```diff
@@ -118,6 +118,10 @@ def method_wrapper(
                 if isinstance(response, list) and n_results is not None:
                     # make sure response has n_results items
                     response = (response * (n_results // len(response) + 1))[:n_results]
+                elif n_results is not None:
+                    # force single response in example into result list (not all examples provide lists)
```
Comment (lwasser) [inline stravalib/tests/integration/strava_api_stub.py:122]: This is super nitpicky @jsamoocha BUT [PEP 8](https://peps.python.org/pep-0008/#comments) specifies that comments should start with a capital letter.  i'd love us to follow pep 8 for consistency!

File: stravalib/tests/integration/strava_api_stub.py [hunk 2]
```diff
@@ -118,6 +118,10 @@ def method_wrapper(
                 if isinstance(response, list) and n_results is not None:
                     # make sure response has n_results items
```
Comment (lwasser) [inline stravalib/tests/integration/strava_api_stub.py:119]:
```suggestion
                    # Make sure response has n_results items
```

**Comment by lwasser** [comment]
just a few TINY syntax corrections - see inline comments and then feel free to merge!

File: stravalib/tests/integration/strava_api_stub.py (hunk 1 above)
Comment (jsamoocha) [reply stravalib/tests/integration/strava_api_stub.py]: Sure, I applied your suggestions; thanks!

**Comment by jsamoocha** [comment]
Thanks for the review! I was able to see the CodeCov report but got an error trying to go into details:
Codecov

I guess it's because we added some commits. I didn't need any credentials to get to that point. Let's see at the next PR if it works (I guess (again) it will because this is a public repo).

**Pull request merged**